packages/chat-client/src/controllers/engine.ts

```diff
@@ -242,6 +242,22 @@ export class ChatEngine extends IChatEngine {
 
     await this.subscribeToSelfInviteTopic();
 
+    if (this.client.syncClient) {
+      try {
+        const { signature } = this.client.syncClient.signatures.get(account);
```

Comment:
The cleaner way to do this seems to be to check for `account` first in the signatures `Store`, otherwise even with this `try/catch` structure we still get an internal error in the signatures `Store` on the `get` every time like we've seen before with this pattern:

```suggestion
if (this.client.syncClient.signatures.keys.includes(account) {
  const { signature } = this.client.syncClient.signatures.get(account);
} else {
  const syncMessage = await this.client.syncClient.getMessage({
    account,
  });
  // ...
}
```

Reply:
Really we should just add `Store.has()` for this but that's another story.